Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -950,433 +909,438 @@
-    <RegEx find="(avgust)(a|e|i|u|om|ima)?" replaceWith="kolovoz$2" />
-    <RegEx find="(avijon)(i|a|e|u|ima)" replaceWith="avion$2" />
-    <RegEx find="(b|B)ekstv(a|u|om)" replaceWith="$1bijeg$2" />
+    <RegEx find="([0-9])-ogodišnj(i|a|e|u)\b" replaceWith="$1-godišnj$2" />
+    <RegEx find="(jeda|dva|tri|četr|pet|šes|sedam|osam|devet)najst(a|e|i|o|u|im|om|og|oj|ima)\b" replaceWith="$1naest$2" />
+    <RegEx find="\b([aA])bsorbira(m|š|mo|te|ju|li|le|la|lo|ti|jući)?" replaceWith="$1psorbira$2" />
+    <RegEx find="\b([aA])bstraktn(a|e|i|o|u|oj|om|og|im|ima)?" replaceWith="$1pstraktn$2" />
+    <RegEx find="\badvokat(a|e|u)?\b" replaceWith="odvjetnik$1" />
+    <RegEx find="\badvokat(i|ima)\b" replaceWith="odvjetnic$1" />
+    <RegEx find="akcion(i|a|e|u|om|im|ima)" replaceWith="akcijsk$1" />
+    <RegEx find="aktuel(an|ni|na|ne|no|nom|nu|nim|nima)" replaceWith="aktual$1" />
+    <RegEx find="[aA]las(ka|ku|ki|ci)" replaceWith="Aljas$1" />
+    <RegEx find="asvalt(a|u|om|ira|irati|iranje|iran|iranog|irano|iranu|iranoj|iranom|na|ni|noj|nom)?" replaceWith="asfalt$1" />
+    <RegEx find="\bavgust(a|e|i|u|om|ima)?\b" replaceWith="kolovoz$1" />
+    <RegEx find="\bavijon(i|a|e|u|ima)\b" replaceWith="avion$1" />
+    <RegEx find="\b([bB])ekstv(a|u|om)\b" replaceWith="$1ijeg$2" />
     <RegEx find="bezbed(an|na|no|ni|nima|nu|ne|nog|nom|noj)" replaceWith="sigur$1" />
     <RegEx find="bezbedno(st|stima|šću|snu|snog|snom|sni)?" replaceWith="sigurno$1" />
     <RegEx find="bakcil(i|a|e|u|ima)?" replaceWith="bacil$1" />
-    <RegEx find="[Bb]iro(a|i|u|om|ima)?" replaceWith="Ured$1" />
-    <RegEx find="(b|B)edn(a|e|i|o|u|om|og|ima)?" replaceWith="$1ijedn$2" />
-    <RegEx find="(b|B)elešk(a|e|u|om|ama)?" replaceWith="$1ilješk$2" />
-    <RegEx find="(b|B)j?esn(i|o|u|om|og|om|oj)" replaceWith="$1ijesn$2" />
-    <RegEx find="(b|B)eznadež(an|na|no|nom|noj|nim|nima)" replaceWith="$1eznad$2" />
-    <RegEx find="(b|B)ež(i|imo|e|ao|ala|ali|anje|anjem|anju)" replaceWith="$1jež$2" />
-    <RegEx find="(b|B)ijež(i|imo|e|ao|ala|ali)" replaceWith="$1jež$2" />
-    <RegEx find="(bioskop)(a|u|om)" replaceWith="kin$2" />
-    <RegEx find="(b|B)(led)(a|e|i|o|u|im|om|ima)" replaceWith="$1lijed$3" />
-    <RegEx find="([bB])(ožiji)(a|i|u|om|ima)?" replaceWith="$1ožji$3" />
-    <RegEx find="(božić)(a|e|i|u|em|ima)?" replaceWith="Božić$2" />
+    <RegEx find="\b[bB]iblij(a|i|u|om|ama)\b" replaceWith="Biblij$1" />
+    <RegEx find="[bB]iro(a|i|u|om|ima)?" replaceWith="Ured$1" />
+    <RegEx find="([bB])edn(a|e|i|o|u|om|og|ima)?" replaceWith="$1ijedn$2" />
+    <RegEx find="([bB])elešk(a|e|u|om|ama)?" replaceWith="$1ilješk$2" />
+    <RegEx find="([bB])j?esn(i|o|u|om|og|om|oj)" replaceWith="$1ijesn$2" />
+    <RegEx find="([bB])eznadež(an|na|no|nom|noj|nim|nima)" replaceWith="$1eznad$2" />
+    <RegEx find="([bB])ež(i|imo|e|ao|ala|ali|anje|anjem|anju)" replaceWith="$1jež$2" />
+    <RegEx find="([bB])ijež(i|imo|e|ao|ala|ali)" replaceWith="$1jež$2" />
```

Comment:
it goes:

`<RegEx find="([bB])i?j?ež(i|imo|e|ao|ala|ali)" replaceWith="$1jež$2" />`